### PR TITLE
Clear rankings with slash command

### PR DESCRIPTION
**Pull request by atomdmac**
# Notes
This does **not** currently ask the user to confirm before clearing rankings. UX-wise, I don't think this is a great way to go but code-as-implemented does work.  That said, I could be swayed either way.

# TODO
  - Add a slash command [here](https://api.slack.com/apps/A9M8Z9LC9/slash-commands?) that points to the `/clear-rankings` endpoint implemented by this PR.

@SharpNotions/ten-hour-project

### CODE REVIEW

**Comment by tstodter** [comment]
Just spit-balling ideas, we could have the actual slash command return a button, that you'd then need to press to confirm the api call.

**Comment by atomdmac** [comment]
@tstodter yeah, that's what I had in mind, too.  If I have enough time this week, I'm hoping to work that in but I wanted to get something that was at least usable out first.

File: handlers/commands/clearRankings.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    send(res, 400, {
+      message: 'user_name and team_domain parameters are required'
+    })
+  }
+
+  if (user_name !== ADMIN_USER_NAME) {
+    send(res, 400, {
+      message: 'INTRUDER!  You\'re not Greg!'
+    })
+  }
```
Comment (ryanSN) [inline handlers/commands/clearRankings.js:33]: 👏

**Review by payers1 — APPROVED** [review]
🚢

**Pull request merged**